scripts/data/updateWidgetsOverview.ts

```diff
@@ -111,3 +137,13 @@ async function extractTextFromFile<P extends Patterns>(
         return value ? { ...result, [key]: value } : result;
     }, {} as Values<P>);
 }
+
+function moduleExports(fileName: string, exportName: string): boolean {
+    const program = ts.createProgram([fileName], {});
```

Comment:
Can we just import the necessary functions instead of the default from "typescript"?